_includes/content/ctl-kubectl-plugin-install.md

```diff
@@ -0,0 +1,41 @@
+## Install calicoctl as a kubectl plugin on a single host
+
+1. Log in to the host, open a terminal prompt, and navigate to the location where
+you want to install the binary.
+
+   > **Note**: In order to install `{{include.cli}}` as a kubectl plugin, the binary must be located in your `PATH`. For example,
+   > `/usr/local/bin/`.
+   {: .alert .alert-info}
+
+1. Use the following command to download the `calicoctl` binary.
+
+   ```bash
+   curl -o kubectl-calico -L  {% include urls component="calicoctl" %}
```

Comment:
this renders to `curl -o kubectl-calico -L  https://projectcalico.org/builds/calicoctl` and that path does not exist.

Reply:
Yeah, it doesn't exist for master. That URL is a placeholder on master but resolves to the correct thing for release branches...

This is the same URL used for our binary download method, for example.

Reply:
Sounds like we need a new post-release check for this binary.

Reply:
maybe, or we just need to change the URL so it doesn't look so "real".

e.g., `https://placeholder-for-calicoctl`

Reply:
Well, where ever it comes from, we need a post-release check for this new release artifact :)

Reply:
Oh no, it's not a new release artifact. It's the same calicoctl binary and URL. We're not introducing anything new here.